udata/models/extras_fields.py

```diff
@@ -37,8 +47,8 @@ def validate(self, value):
                      else extra_cls(**value).validate())
                 else:
                     extra_cls().validate(value)
-            except ValidationError as e:
-                errors[key] = e.message
+            except Exception as e:
```

Comment:
What was wrong w/ `ValidationError`? Seemed safer.

Reply:
Because there are other exceptions raised depending on the used field :/

Reply:
Can we list them? I feel like this is the kind of code that can catch a totally unrelated exception from way back and make it difficult to debug... Like a unicode mayhem for example.

Reply:
Not exactly, the `extra_cls` is dynamic, listing exceptions means handling every case of every known and unkown fields.
But don't worry, the exception is serialized into the errors on the approriate key, they are only validation errors

Reply:
I'm still worried, but OK ;-)

Reply:
OK, I'll try to immrove that in another PR

Reply:
OK, I narrowed the catched exeptions